addOns/replacer/src/main/java/org/zaproxy/zap/extension/replacer/OptionsReplacerPanel.java

```diff
@@ -43,10 +43,10 @@
 
     public OptionsReplacerPanel() {
         super();
-        initialize();
+        init();
```

Comment:
Or remove.

Reply:
Do you mean move all the initialization content into the constructor? (Same below).

Reply:
Yes, it does not need to be "moved" though, deleting the method declaration is enough to be in the constructor.

Reply:
Okay so I just pushed this change, let me know if we're on the same page.

Reply:
We are :)